src/microbit/pxt_gangespill/gangespill.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "PXT: Gangespill"
+level: 1
+author: Espen Clausen og Tjerand Silde
+language: nb
+---
+
+# Introduksjon {.intro}
+
+Dette er et enkelt og modifiserbart gangespill for micro:bit. Det fungerer slik at 
+når en rister micro:biten så vil en få presentert et gangestykke. En svarer ved å trykke på 
+A-knappen for antall tiere i svaret og B-knappen for antall enere. Svarer en korrekt 
+vil en få opp et smilefjes, hvis ikke kommer et surt fjes.
+
+# Steg 1: Når microbit ristes {.activity}
+
+## Sjekkliste {.check}
+
+- [ ] Start et nytt PXT-prosjekt, for eksempel ved å gå til
+  [makecode.microbit.org](https://makecode.microbit.org/?lang=no).
+
+- [ ] Vi vil at noe skal skje når vi rister på micro:biten. Til dette kan vi
+  bruke `når ristes`-klossen som finnes i kategorien `Inndata`.
+
+- [ ] Når micro:bit ristes skal det vises et gangestykke på skjermen. Vi må opprette en 
+	  del `variabler` til å ta vare på verdiene vi skal bruke underveis: `_10er` tar vare på 
+	  antall tiere i svaret, `_1er` tar vare på antall enere i svaret, `svar` er summen av 
+	  variablene 10er og 1er, mens `produkt` er svaret som micro:bit regner seg fram til. 
```

Comment:
`_navn` er konvensjon for private variabler / spesialvariabler i Python. Foreslår `enere` og `tiere` i stedet. [Den offisielle stilguiden](https://www.python.org/dev/peps/pep-0008/) er dog ikke helt bestemt på dette.